projects/storefrontapp/server.ts

```diff
@@ -5,11 +5,11 @@
  */
 
 import { APP_BASE_HREF } from '@angular/common';
-import { ngExpressEngine as engine } from '@nguniversal/express-engine';
 import {
-  NgExpressEngineDecorator,
+  CxExpressEngineDecorator,
   SsrOptimizationOptions,
   defaultSsrOptimizationOptions,
+  cxExpressEngine as engine,
```

Comment:
[MAJOR]
I guess we longer need the `ngExpressEngine` to be passed to some other function that decorates it, because now we control and create the `ngExpressEngine`. The decorator was meant to wrap the original Angular's `ngExpressEngine` that we don't create.

That said, removing it would be yet another migration to provide.

Example, but let's not do it now, only if we have more time:
```ts
import { createNgExpressEngine } from '@spartacus/setup/ssr;

const ngExpressEngine = createNgExpressEngine(ssrOptions};
```